### DISCUSSION TITLE
How do you enable this extension for `git commit` messages?

### DISCUSSION THREAD

**Discussion by hirehamir** [streetsidesoftware/vscode-spell-checker#3227] [Q&A] [answered]
I tried add `COMMIT_EDITMSG` to `cSpell.enableFiletypes` but that didn't seem to work.

**Comment by hirehamir** [comment]
# Screen recording of issue

[Screencast from 05-05-2024 01:01:14 AM.webm](https://github.com/streetsidesoftware/vscode-spell-checker/assets/43425812/be5de9f3-a90b-49cb-a31f-77b0c59d359f)

**Comment by Jason3S** [comment, marked as answer]
@hamirmahal,

It is always a bit of a mystery the name VS Code uses for a file type, and they change between releases of VS Code. To find out:

1. Open the file in question (`.git/COMMIT_EDITMSG` in your case).
2. Hover over `Spell` on the status bar: <img width="58" alt="image" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/86e4e1e7-710a-4e4e-bb40-93a2e4f03d79">
   <img width="397" alt="Screenshot 2024-05-05 at 13 39 13" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/fa6c73ca-d4d7-45fc-a871-43712fb5d904">

3. The hover message shows the reasons it is not spell checked. `git-commit` is the file type needed and you might need to edit your configuration if `.git` has been excluded in the `ignorePaths`.

**Reply by hirehamir** [reply]
Thank you for the help! And thank you for responding so quickly.
